js/script.js

```diff
@@ -34,36 +36,42 @@ class ToDoList{
         return answer;
     }
     notify(msg, elementRef){
-        elementRef?elementRef.style.cssText = "border-bottom: 1px solid red":false;
+        elementRef ? elementRef.style.cssText = "border-bottom: 1px solid red" : false;
         this.msgEl.innerText = msg;
     }
     add(){
         let toDoList = this.getData();
         const toDo = {};
+
         if(toDoList){
             const lastId = toDoList[toDoList.length - 1].id;
             toDo.id = lastId + 1;
             toDo.name = this.inputText.value;
             toDo.done = false;
         }
+
         else{
             toDoList = [];
             toDo.id = 1;
             toDo.name = this.inputText.value;
             toDo.done = false;
         }
+
         const validToDo = this.validate(toDo.name);
 
         if(validToDo === true) {
```

Comment:
return the object with the desired flag and message, if required, but it's like a crutch